src/styles/utilities/_responsive.css

```diff
@@ -5,8 +5,8 @@
 }
 
 .u-hidden-mobile {
-  display: none;
+  display: none !important;
```

Comment:
Here the usage of !importatnt is completely valid. 
Concept taken from bootstrap 
![image](https://github.com/aeternity/aescan/assets/15363559/7fee561b-94ae-4acf-810c-735f932a24a3)